src/components/07-form/templates/address-form.njk

```diff
@@ -17,7 +18,7 @@
         <div class="usa-input-grid usa-input-grid-small">
           <label for="state">State</label>
           <select id="state" name="state">
-            <option value></option>
+            <option value>- Select -</option>
```

Comment:
does this need to be set to `option value=""`? I saw that's what it was in a different PR for the `dropdown.njk` file and I'm not sure which is right.